plugins/source/alicloud/resources/services/bss/bill_des.go

```diff
@@ -0,0 +1,57 @@
+package bss
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/alicloud/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func Bill_Des() *schema.Table {
+	return &schema.Table{
+		Name:      "alicloud_bss_bill_des",
+		Resolver:  fetchBssDescribeinstanceBill,
+		Multiplex: client.AccountMultiplex,
+		Transform: transformers.TransformWithStruct(
+			&BillDesModel{},
+			transformers.WithPrimaryKeys(
+				"BillingCycle",
```

Comment:
As you unwrap items from response, several of those will have the same `BillingCycle`.
Could you check if there's a better PK to be used?